### PR TITLE
doc: fix description of the `PERFORMANCE_PROFILE_INPUT_FILES` env var

### PR DESCRIPTION
**Pull request by jlojosnegros**
Environment variable `PERFORMANCE_PROFILE_INPUT_FILES` used in render mode
should be a comma separated list of PerformanceProfile manifests not a
folder.

### CODE REVIEW

File: docs/performanceprofile/performance_addon.md [hunk 1]
```diff
@@ -37,72 +37,74 @@ The deployment will be retried in a loop until everything is deployed successful
 In CI the `test/e2e/performanceprofile/cluster-setup/ci-cluster/performance/` dir will be used. The difference is that the CI cluster will deploy
 the PerformanceProfile in the test code, while the `manual` cluster includes it in the kustomize based deployment.
 
-
 Now you need to label the nodes which should be tuned. This can be done with
 
-```
+```cli
```
Comment (MarSik) [inline docs/performanceprofile/performance_addon.md:42]: Why cli here vs. shell in the two other changes, is that intentional?

File: docs/performanceprofile/performance_addon.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 The `Performance Addon controller` optimizes OpenShift clusters for applications sensitive to cpu and network latency.
 
-![alt text](https://github.com/openshift/cluster-node-tuning-operator/blob/master/docs/performanceprofile/interactions/diagram.png "How Performance Addon controller interacts with other components and operators")
+!["How Performance Addon controller interacts with other components and operators"](https://github.com/openshift/cluster-node-tuning-operator/blob/master/docs/performanceprofile/interactions/diagram.png )
```
Comment (marioferh) [inline docs/performanceprofile/performance_addon.md:5]: Performance Addon Controller. I thin the lower case c is a typo from old commit.

**Review by marioferh — COMMENTED** [review]
/lgtm

File: docs/performanceprofile/performance_addon.md (hunk 1 above)
Comment (jlojosnegros) [reply docs/performanceprofile/performance_addon.md:42]: Nope, just trying to see which one renders better. 
I will change it to `shell` as there is no difference .

File: docs/performanceprofile/performance_addon.md (hunk 2 above)
Comment (jlojosnegros) [reply docs/performanceprofile/performance_addon.md:5]: done

File: docs/performanceprofile/performance_addon.md (hunk 1 above)
Comment (jlojosnegros) [reply docs/performanceprofile/performance_addon.md:42]: done

**Comment by jlojosnegros** [comment]
just rebasing

**Review by marioferh — COMMENTED** [review]
/lgtm

**Comment by fromanirh** [comment]
/retitle doc: fix description of the `PERFORMANCE_PROFILE_INPUT_FILES` env var

**Pull request merged**